heron/common/src/java/com/twitter/heron/common/basics/FileUtils.java

```diff
@@ -123,4 +123,17 @@ public static boolean isOriginalPackagePex(String packageFilename) {
   public static String getBaseName(String file) {
     return new File(file).getName();
   }
+
+  public static String getPkgType(String topologyBinaryFile) {
```

Comment:
where do you think is appropriate to put this? I am really bad at finding an appropriate class to add code😔